CustomSecuritySample/Setup/RestoreDatabase.sql

```diff
@@ -0,0 +1,22 @@
+/****************************************************************************
+  This file is part of the Microsoft SQL Server Code Samples.
```

Comment:
Changes done to the respective files. Did a quick check on other files it's been updated to MIT License.